tests/test_examples.py

```diff
@@ -16,26 +16,34 @@
 
 from gordo_components.data_provider.providers import DataLakeProvider
 from gordo_components.dataset.datasets import TimeSeriesDataset
+from gordo_components.dataset.sensor_tag import SensorTag
 
 logger = logging.getLogger(__name__)
 
+CONFIG = dict(
+    from_ts=dateutil.parser.isoparse("2014-07-01T00:10:00+00:00"),
+    to_ts=dateutil.parser.isoparse("2015-01-01T00:00:00+00:00"),
```

Comment:
Should rather be `train_start/end_date` the only reason they still work with `TimeSeriesDataset` is because we have a compatibility decorator on it's `init`